src/nystrum/entites.js

```diff
@@ -899,11 +902,19 @@ const Dragging = superclass => class extends superclass {
 }
 
 const Draggable = superclass => class extends superclass {
-  constructor({ ...args }) {
+  constructor({draggability = true, ...args }) {
     super({ ...args })
+    this.draggability = draggability
     this.entityTypes = this.entityTypes.concat('DRAGGABLE')
   }
 }
+const Pushable = superclass => class extends superclass {
+  constructor({pushability = true, ...args}) {
```

Comment:
since pushability and draggability are booleans, I think it'd read better to rename these variables as 'draggable' and 'pushable'.